documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-application-credential-authentication_{context}"]
+= Using application credential authentication with an {osp} source provider
+
+You can use application credential authentication, instead of username and password authentication, when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
```

Comment:
```suggestion
You can use application credential authentication, instead of username and password authentication, when you create an {osp} source provider.
```